addon/templates/components/big-text.hbs

```diff
@@ -1,8 +1,16 @@
-<span class={{if renderHtml "list-reset"}}>
+<span
+  class="editable {{if renderHtml "list-reset"}}"
```

Comment:
Is `editable` needed here now that `[data-test-edit]` is used for tests?

Reply:
Once I removed `editable`, I think bunch of other related tests were breaking. I could further fix those if you want.

Reply:
Editable wasn't part of this element originally, so if other tests are breaking when it is removed then they should be fixed as well.

Reply:
I can look into it. `editable` class was part of the overall `span` before I refactored it so other tests were still depending on that class.

Reply:
Fixed.

Reply:
I still see `editable` here.

Reply:
Done.